src/Spd.Resource.Repository/Document/DocumentRepository.cs

```diff
@@ -40,7 +40,10 @@ public async Task<DocumentListResp> QueryAsync(DocumentQry qry, CancellationToke
     {
         var documents = _context.bcgov_documenturls.Where(d => d.statecode != DynamicsConstants.StateCode_Inactive);
         if (qry.ApplicantId != null)
-            documents = documents.Where(d => d._spd_submittedbyid_value == qry.ApplicantId);
+            documents = documents.Where(d => d._bcgov_customer_value == qry.ApplicantId);

```

Comment:
do we know this change would effect on other apps that have reference to this function?
there is 46 refs

Reply:
yeah, actually, the old code is wrong.